LuaMenu/widgets/gui_replay_handler.lua

```diff
@@ -3,12 +3,12 @@
 
 function widget:GetInfo()
 	return {
-		name      = "Replays window",
-		desc      = "Handles local replays.",
-		author    = "GoogleFrog",
```

Comment:
Im pretty sure you now deserve a mention here

Reply:
I admit I didn't even think about that ^^ I'm in the commit history, what more could you ask for ? 

Since there is only one author, I don't know what the convention is TBH

Reply:
Comma separated list of people who worked on this :)